### PR TITLE
Silent/Penitent Girl

### PR DESCRIPTION
**Pull request by dsmith328**
## About The Pull Request
Adds Silent Girl and her EGO.
Adds the Universal Work Modifier.
Guarantees a White Lake champion will never be buffed twice by resetting the variable.
Adds Work Complete and Work Start signals.
Changes Fairy Festival to use these signals instead.

## Why It's Good For The Game
More HE Good. White Resistance is very nice earlier on. Adds a nice WAW HE weapon with decent requirements. Helps farm but can not be farmed recklessly. Signals can also be used for a NUMBER of different abnormality's functionality.

## Changelog
:cl:
added silent_girl.dm to code\module\mob\living\simple_animal\abnormality\he
added code to code\_DEFINES\dcs\signals.dm
added code to code\datums\abnormalities\_ego_datum\waw.dm
added code to code\datums\abnormalities\_ego_datum\he.dm
added code to code\modules\clothing\suits\ego_gear\he.dm
added code to code\module\mob\living\simple_animal\abnormality\he\white_lake.dm
altered the signals in Fairy Festival, literally just change the signal.
added code to code\game\objects\items\ego_weapons\waw.dm
added code to code\modules\mob\living\carbon\human\physiology.dm
added code to code\game\machinry\computer\abnormality_work.dm
/:cl:

### CODE REVIEW

**Comment by EgorDinamit** [comment]
I believe fairy festival PR already added signals for work attempts, you should use these instead, no?

File: code/game/machinery/computer/abnormality_work.dm [hunk 1]
```diff
@@ -109,6 +109,10 @@
 	var/work_chance = datum_reference.get_work_chance(work_type, user)
 	work_chance *= 1 + (get_attribute_level(user, TEMPERANCE_ATTRIBUTE) / 180)
 	work_chance = datum_reference.current.work_chance(user, work_chance)
+	if (user.physiology.work_success_mod <= 0)
+		work_chance = 0
+	else
+		work_chance *= user.physiology.work_success_mod
```
Comment (EgorDinamit) [inline code/game/machinery/computer/abnormality_work.dm:115]:
```suggestion
	work_chance = clamp(work_chance * user.physiology.work_success_mod, 0, 100)
```

**Comment by EgorDinamit** [comment]
Merge conflicts have arrived.

**Comment by dsmith328** [comment]
I
I'm so fucking confused how files I don't even touch get changed

**Pull request closed without merging**